andela/krypton-ah-frontend
# #162414128 Reducers to update read article page with total number of likes and dislikes

**What does this PR do?**
 Update read article page with the total number of likes and dislikes

**Description of Task to be completed?**
- Fetch article reaction data from database and update state

#### How should this be manually tested?
Git clone the repo and cd into it
`npm run dev `to start the server
click on any article on the homepage to read
scroll down to the end of the article and the number of likes and dislikes will be displayed
open the redux dev tools to check the application state

#### What are the relevant pivotal tracker stories?
[#162414128](https://www.pivotaltracker.com/story/show/162414128)

**Screenshots (if appropriate)**
![likereducers](https://user-images.githubusercontent.com/22124878/53506429-3b9b4080-3ab6-11e9-96e9-110526589eae.gif)
